lib/ts/utils/windowHandler/defaultImplementation.ts

```diff
@@ -134,6 +134,9 @@ export const defaultWindowHandlerImplementation: WindowHandlerInterface = {
     getDocument: function() {
         return getWindowOrThrow().document;
     },
+    getWindowUnsafe: function() {
+        return getWindowOrThrow().window;
+    },
```

Comment:
we need to check what happens is auth-react is used in hybrid apps like:
- capacitor
- electron

Reply:
Checked both, and it worked without any issues.